lib/nodeVersion.js

```diff
@@ -0,0 +1,32 @@
+const debugLog = require('debug')('lint-staged:node-version')
+const semverSatisfies = require('semver/functions/satisfies')
```

Comment:
Since this was the beef of `please-upgrade-node`, I opted for smaller footprint.